packages/sit-onyx/src/components/OnyxColorSchemeMenuItem/OnyxColorSchemeMenuItem.ct.tsx

```diff
@@ -0,0 +1,52 @@
+import { expect, test } from "../../playwright/a11y";
+import { executeMatrixScreenshotTest } from "../../playwright/screenshots";
+import type { ColorSchemeValue } from "../OnyxColorSchemeDialog/types";
+import OnyxColorSchemeMenuItem from "./OnyxColorSchemeMenuItem.vue";
+
+test.describe("Screenshot tests", () => {
+  executeMatrixScreenshotTest({
+    name: "Color scheme menu item",
+    columns: ["default"],
+    rows: ["default"],
+    // TODO: remove when contrast issues are fixed in https://github.com/SchwarzIT/onyx/issues/410
+    disabledAccessibilityRules: ["color-contrast"],
+    component: () => (
+      <ul style={{ listStyle: "none", padding: 0 }}>
+        <OnyxColorSchemeMenuItem modelValue="auto" />
+      </ul>
+    ),
+  });
+});
+
+test("should behave correctly", async ({ page, mount }) => {
+  const modelValueEvents: ColorSchemeValue[] = [];
+
+  // ARRANGE
+  const component = await mount(OnyxColorSchemeMenuItem, {
+    props: {
+      modelValue: "auto",
+    },
+    on: {
+      "update:modelValue": (value: ColorSchemeValue) => modelValueEvents.push(value),
+    },
+  });
+
+  // ASSERT
+  await expect(component).toContainText("Appearance: Auto");
+
+  // ACT
+  await component.click();
+
+  // ASSERT
+  const dialog = page.getByRole("dialog");
+  await expect(dialog).toBeVisible();
+
+  // ACT
+  await dialog.getByLabel("Light").click({ force: true });
```

Comment:
This seems to work just aswell and does not need force
```suggestion
  await dialog.getByRole('heading', { name: 'Light' }).click();
```